src/Tests/Microsoft.Diagnostics.Monitoring.ConfigurationSchema/SchemaGenerator.cs

```diff
@@ -29,11 +38,64 @@ public string GenerateSchema()
                 kvp.Value.Default = JsonSchema.CreateAnySchema();
             }
 
+            JsonSchema jsonConsoleLoggerOptionsSchema = GenerateConsoleLoggerOptionsSchema(jsonConsoleFormatterOptions, ConsoleLoggerFormat.Json);
```

Comment:
A few lines above this comment, there is code to add an empty object schema for a handful of egress properties. You'll have to do the same for the following properties in order to get better defaults for object-typed properties:

- LoggingOptions.LogLevel
- LoggingOptions.Console
- LoggingOptions.EventLog
- LoggingOptions.Debug
- LoggingOptions.EventSource
- LogLevelOptions.LogLevel
- ConsoleLoggerOptions.FormatterOptions
- ConsoleLoggerOptions.LogLevel (when you add this to ConsoleLoggerOptions)

Otherwise, you get this which isn't helpful in determining what type of property it is:
![nulldefault](https://user-images.githubusercontent.com/17344677/128537714-d0937fca-a70c-4b8c-aae9-ec0c30ad8e77.png)